src/styles/mixins.scss

```diff
@@ -48,18 +48,18 @@
   transform: translateX($px);
 }
 
-@mixin btn($color, $border-color, $cursor: pointer, $opacity: 1, $hover: true) {
-  cursor: $cursor!important;
+@mixin btn($color, $border-color, $cursor: pointer, $opacity: 1, $hover-fx: true) {
+  cursor: $cursor;
```

Comment:
removed `!important` to allow btn-inactive to maintain non-hover style on _mouseover_.
/cc @codegaze